aperturedb/Images.py

```diff
@@ -45,17 +45,30 @@ class Images(Entities):
     def inspect(self, use_thumbnails=True) -> Union[Tuple[widgets.IntSlider, widgets.Output], DataFrame]:
         df = super(Images, self).inspect()
         if use_thumbnails:
+            def image_base64(im):
+                with BytesIO() as buffer:
+                    im.save(buffer, 'jpeg')
+                    return base64.b64encode(buffer.getvalue()).decode()
+
+            def get_formatters(width):
+                def image_formatter(im):
+                    return f'<img width={width} style="max-width: 400px" src="data:image/jpeg;base64,{image_base64(im)}" >'
+
+                return image_formatter
+
             sizer = widgets.IntSlider(min=1, max=400, value=100)
             op = widgets.Output()
 
-            def widget_interaction(c):
-                def image_base64(im):
-                    with BytesIO() as buffer:
-                        im.save(buffer, 'jpeg')
-                        return base64.b64encode(buffer.getvalue()).decode()
+            with op:
+                image_formatter = get_formatters(sizer.value)
+                display(HTML("<div style='max-width: 100%; overflow: auto;'>" +
+                             df.to_html(
+                                 formatters={'thumbnail': image_formatter}, escape=False)
+                             + "</div>")
+                        )
 
-                def image_formatter(im):
-                    return f'<img width={c["new"]} style="max-width: 400px" src="data:image/jpeg;base64,{image_base64(im)}" >'
+            def widget_interaction(c):
```

Comment:
it's a little tricky to figure out what changed here - it looks like image_formatter became a function that is returned by get_formatters rather than one passed directly? was the img width not updating?

Reply:
We've had a smallish widget for notebook . Its used in this [example](https://github.com/aperture-data/demos/blob/badf7f0d530b06321ac2f287392c5cfdab59c095/ml_notebooks/Vector-based%20Queries.ipynb). Refer to end of 6'th cell.

It had a problem on first render. This addresses it.